README.md

```diff
@@ -145,10 +145,37 @@ $ cat /tmp/20200710182129_ksvc_creation_time.csv
   ktest-109,ktest-1,13,13,2,0,0,0,0,15,2,0,2,15
   ktest-110,ktest-1,14,13,1,0,0,0,0,18,3,0,3,18
 ```
-**Sample Dashboard**
+
+### As a Knative developer, I want to have a kperf dashboard to visualize the measurement result
+
+A visualized result is automatically generated by kperf during the measurement step to make the measurement data to be intuitive, which is a static HTML file including a chart and a table.
+
 ![service_creation_duration measurement](docs/kperf_dashboard.png)
 
+As we will have tons of measurement metrics for a single test (e.g. `revision_ready`, `configuartion_ready`, `ingress_ready` and etc.), it’s important to pick some key measurement metrics to analyze the bottleneck of the Knative service creation lifecycle.
+
+The kperf dashboard is enabled to select or unselect the metrics to be observed, makes it much easier for developers to find out the most time consuming operations during the service creation. Besides, the kperf dashboard is supported to zoom in/out the data, developers can choose the full set or only a subset of the data to observe.
```

Comment:
“the kperf dashboard is supported ...” => “the kperf dashboard supports zooming in/out”